scripts/test.ts

```diff
@@ -0,0 +1,97 @@
+import axios from "axios";
+import { config as dotenvConfig } from "dotenv";
+import { ethers } from "ethers";
+import { resolve } from "path";
+
+import { abi as VolOracleABI } from "../artifacts/contracts/VolOracle.sol/VolOracle.json";
+
+const dotenvConfigPath: string = "../.env";
+dotenvConfig({ path: resolve(__dirname, dotenvConfigPath) });
+
+const alchemyApiKey: string | undefined = process.env.ALCHEMY_API_KEY;
+if (!alchemyApiKey) {
+  throw new Error("Please set your ALCHEMY_API_KEY in a .env file");
+}
+
+const provider = new ethers.providers.JsonRpcProvider(`https://polygon-mainnet.g.alchemy.com/v2/${alchemyApiKey}`);
```

Comment:
no need to initialize here, u can get this infro from `hardhat.config.ts`

Reply:
I'm leaning towards keeping this separated from hardhat.config.ts for simplicity and avoid extra dependencies. In the future this may evolve into SDK